### PR TITLE
Mcs 202: Generate Two Separate Scene Configuration Files

### PR DESCRIPTION
**Pull request by chrislong**

### CODE REVIEW

File: scene_generator/scene_generator.py [hunk 1]
```diff
@@ -65,7 +66,8 @@ def generate_file(name, goal_type, find_path):
     goal_obj = goals.choose_goal(goal_type)
     goal_obj.update_body(body, find_path)
 
-    write_scene(name + '-debug', body)
+    debug_name = name[:-5] + '-debug.json'
```
Comment (deanwetherby) [inline scene_generator/scene_generator.py:83]: Could use pathlib stem  or os.path.splitext here to get the file basename.

Comment (chrislong) [reply scene_generator/scene_generator.py:83]: Yeah, normally I would do something like that, but in this case it's required to end in `.json`.

File: scene_generator/scene_generator.py [hunk 2]
```diff
@@ -40,12 +40,26 @@
...
+
+
 def clean_object(obj):
     """Remove properties we do not want TA1s to have access to."""
+    obj.pop('info', None)
     obj.pop('dimensions', None)
     obj.pop('intphys_option', None)
     if 'shows' in obj:
-        obj['shows'].pop('bounding_box', None)
+        obj['shows'][0].pop('bounding_box', None)
```
Comment (brianpippin) [inline scene_generator/scene_generator.py:62]: This change is another code review I think?

Comment (chrislong) [reply scene_generator/scene_generator.py:62]: Yeah, it is. I wanted the tests to pass.

**Pull request merged**